Sources/FioriAR/ARCards/Views/Authoring/SceneAuthoringController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SceneAuthoringController.swift
+//
+//
+//  Created by O'Brien, Patrick on 12/16/21.
+//
+
+import SAPFoundation
+import SwiftUI
+
+/**
+ Provides the flow for authoring an AR Annotation Scene using UIKit. Used to create the content for the cards, select an anchor Image, and position the entities in their real world locations. Publishing the scene using Mobile Services with an SAPURLSession.
+ The onSceneEdit modifier provides a callback on editing events. When a scene was published, i.e. created or updated in SAP Mobile Services, then `.published(sceneID)` is called and returns the technical id of the scene.
```

Comment:
`onSceneEdit` cannot be used as it returns `some View` but the constraint is `UIHostingViewController<SceneAuthoringView>

Reply:
Very good catch on this as I completely overlooked it. I came to the same conclusion looking for an alternative solution. I usually prefer not using AnyView if I can avoid it, but in this scenario I think the proposal is appropriate.